packages/css/src/components/header/header.scss

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../common/breakpoint";
+
+.amsterdam-header {
+  align-items: center;
+  background-color: var(--amsterdam-header-background-color);
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding-block: 0.88rem;
+  row-gap: 1.5rem;
+
+  @media screen and (max-width: $amsterdam-breakpoint-medium) {
```

Comment:
We could make it easy for ourselves and display the page menu links from on the wide grid only?

Reply:
Only if there is a subsite title?